app/src/main/java/org/connectbot/util/VolumePreference.java

```diff
@@ -56,7 +57,13 @@ protected View onCreateDialogView() {
 		sb.setMax(100);
 		sb.setProgress((int) (getPersistedFloat(
 				PreferenceConstants.DEFAULT_BELL_VOLUME) * 100));
-		sb.setPadding(10, 10, 10, 10);
+
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			sb.setPadding(75, 70, 75, 10);
```

Comment:
Better yet:

getContext().getResources().getDimension(android.support.design.R.dimen.abc_select_dialog_padding_start_material)

and

getContext().getResources().getDimension(android.support.design.R.dimen.abc_dialog_list_padding_vertical_material)